doc/developer/01_getting_started.md

```diff
@@ -0,0 +1,70 @@
+![hitobito logo](https://user-images.githubusercontent.com/9592347/184715060-351453d4-d066-4ff6-8f82-95d3b524b62f.svg)
+## Getting started 🍀🍀🌅
+
+To get started with the project following steps are required:
+
+### Stage 1: Get Hitobito to run locally
+
+- [ ] Setup Project locally, please follow the instructions at [Development](https://github.com/hitobito/development/) 
+for the recommended 🚢 docker development setup.
+
+
+- [ ] Make sure Tests 👨‍🔬 are running, look at [Tests](02_testing)
+
+
+- [ ] Check if you can access the mailcatcher which run on [http://localhost:1080](http://localhost:1080)
+
+
+- [ ] Look 👁️ at the table with all specific rake tasks in it, try some of them if interested [Rake Tasks](#specific-rake-tasks) 
+
+Run the following rake tasks inside rails, rails-test container:
+
+| Task                      | Beschreibung                                                                                           |
+| ------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `rake hitobito:abilities` | Alle Abilities ausgeben.                                                                               |
+| `rake hitobito:roles`     | All Gruppen, Rollen und Permissions ausgeben.                                                          |
+| `rake annotate`           | Spalten Informationen als Kommentar zu ActiveRecord Modellen hinzufügen.                               |
+| `rake rubocop`            | Führt die Rubocop Must Checks (`rubocop-must.yml`) aus und schlägt fehl, falls welche gefunden werden. |
+| `rake rubocop:report`     | Führt die Rubocop Standard Checks (`.rubocop.yml`) aus und generiert einen Report für Jenkins.         |
+| `rake brakeman`           | Führt `brakeman` aus.                                                                                  |
+| `rake mysql`              | Lädt die MySql Test Datenbank Konfiguration für die folgednen Tasks.                                   |
+| `rake license:insert`     | Fügt die Lizenz in alle Dateien ein.                                                                   |
+| `rake license:remove`     | Entfernt die Lizenz aus allen Dateien.                                                                 |
+| `rake license:update`     | Aktualisiert die Lizenz in allen Dateien oder fügt sie neu ein.                                        |
+| `rake ci`                 | Führt die Tasks für einen Commit Build aus.                                                            |
+| `rake ci:nightly`         | Führt die Tasks für einen Nightly Build aus.                                                           |
+| `rake ci:wagon`           | Führt die Tasks für die Wagon Commit Builds aus.                                                       |
+| `rake ci:wagon:nightly`   | Führt die Tasks für die Wagon Nightly Builds aus.                                                      |
+---
+
```

Comment:
create a getting started tutorial ... 
for example how to customize a function in a wagon